tests/test_generators/test_plantuml.py

```diff
@@ -45,7 +45,7 @@ class "Person" [[{A person, living or dead}]] {
 
 @pytest.fixture(scope="module")
 def kroki_url(request):
-    kroki_container = DockerContainer("yuzutech/kroki").with_exposed_ports(8000)
+    kroki_container = DockerContainer("yuzutech/kroki").with_exposed_ports(65432)
```

Comment:
maybe intended for another PR?

Reply:
Not really, only a local requirement, since my VPN blocks lots of ports below 64000. It is a late-night error having pushed it. Will remove it, I still have to clean-up a bit. Especially WRT git history.